### PR TITLE
Okta authjs pkce

### PR DESCRIPTION
**Pull request by dogeared**
This blog post has the following features:

- [x] A GitHub Repository with a polished README
- [x] A title that's approved by Dev Marketing
- [x] A URL that's approved by Dev Marketing
- [x] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2019-08-14-okta-authjs-pkce.md [hunk 1]
```diff
@@ -0,0 +1,334 @@
+---
+layout: blog_post
+title: "Implement the OAuth 2.0 Authorization Code with PKCE flow"
+author: dogeared
```
Comment (mraible) [inline _source/_posts/2019-08-14-okta-authjs-pkce.md]: Capitalize flow

File: _source/_posts/2019-08-14-okta-authjs-pkce.md [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+description: "This tutorial shows you how to migrate from the OAuth 2.0 Implicit flow to the more secure Authorization Code with PKCE flow"
```
Comment (mraible) [inline _source/_posts/2019-08-14-okta-authjs-pkce.md]: Add period at the end of sentence.

File: _source/_posts/2019-08-14-okta-authjs-pkce.md [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+tags: [java, oauth2, tutorial, security, authorization, pkce]
+tweets:
+- "The OAuth 2.0 Implicit flow for SPAs is (effectively) deprecated. @afitnerd shows you how to easily switch to PKCE for secure SPAs"
+- "Implement the OAuth 2.0 Authorization Code with PKCE flow with Javascript and Okta"
```
Comment (mraible) [inline _source/_posts/2019-08-14-okta-authjs-pkce.md]: JavaScript

File: _source/_posts/2019-08-14-okta-authjs-pkce.md [hunk 4]
```diff
@@ -0,0 +1,334 @@
...
+export function loginOkta(grantType) {
+    oktaAuth.options.grantType = grantType;
+    oktaAuth.token.getWithRedirect({
+        responseType: responseTypes[grantType],
+        scopes: ['openid', 'profile', 'email']
+    });
+}
+```
+
+The `loginOkta` function sets the `grantType` and the `responseType` in the `okta-auth-js` library which is how this code supports both flows dynamically.
```
Comment (mraible) [inline _source/_posts/2019-08-14-okta-authjs-pkce.md]: Add () after function name

**Comment by dogeared** [comment]
@mraible - I didn't see your suggestions before I merged (damn lag on a boat). I created a separate branch with your changes which I will merge once CI passes.

I didn't implement the last suggestion since the function takes a param and since the code of function itself comes right after the sentence.

**Pull request merged**